backend/controllers/post.controller.ts

```diff
@@ -140,13 +136,19 @@ export class PostController {
 
     if (user.role === UserRole.INDIVIDUAL && type === PostType.REQUEST) {
       throw new AuthorizationError(
-        `Individual users are unable to create request posts.`
+        `Individual users are unable to create request posts.`,
       );
     }
 
     if (user.active === false) {
       throw new AuthorizationError(
-        `User ${user.username} is deactivated and is not authorized to create a post.`
+        `User ${user.username} is deactivated and is not authorized to create a post.`,
+      );
+    }
+
+    if (user.organization.verified != true) {
```

Comment:
I think we should prefer strict equality in JavaScript.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality

Reply:
Reason I used `!=` is bc I don't know if .verified can take any other value than `true` and `false`. I couldn't quite figure out where it was assigned to `false` initially, i thought it might can take `none` as value

Reply:
I think we can simplify this check by using the logical NOT operator (!), which will convert the object and we can check for truthy/falsy in a more idiomatic way.
`if (!user.organization.verified) {`

Reply:
Ohh ok